From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@raymondlaghaeian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

zhouwangzw committed:
new speech documentation for review

 * Significant changes to improve quality and readablity.
 * Provide very clear and handy GetStart page with intuitive, handy, runnable examples to make it super easy for new comers/unexperienced users to onboard our speech service.
 * Add Concepts, How-to Guides and Troubleshooting pages to help advacned users to pick up right approach, give them guidance, best-practices, and help when facing issues.
 * Fix many out-of-dated content, mal-formed text, fix inconsistencies among pages, fix broken links, improve readablity by re-organizing pages and re-formulating text.

use subscription key in REST; complete how-to-authentication, troubleshooting

complete supportedlanguages, getstartedclientlibraries, how-to-chunked-transfer

add REST samples, fix toc

fix links

add concepts

add redirect of BingVoiceREcognition.md; fix links

fix links, add redirects

fix more errors, made improvement as Acrolinux suggested

fix recommendations from Acrolinx; fix line breaks in websocketprotocol.md

revise websocketprotocol.md

rework concepts and websocketprotocol

add image file

revise formatting, links

fix invalid links

fix internal link reference

update getstartedcsharpdesktop.md, fix links

fix styles

fix styles

more fixes on style and link

use Speech Recognition API

update getstarted for android

update getstarted on ios

fix typo, errors, and style

fix unicode characters

fix more unicodes

fix formatting errors

fix broken links and formatting

minor changes

fix typo

add link to troubleshooting

address CR comments

moderakh committed:
python sdk 2.2.1 release

neilpeterson committed:
added powershell ref

BryanLa committed:
Reformatting

nberdy committed:
Update iot-hub-store-data-in-azure-table-storage.md

rothja committed:
First draft of SQL VM Linux FAQ

rothja committed:
TOC entry for new FAQ topic

nberdy committed:
Add files via upload

nberdy committed:
Add files via upload

nberdy committed:
Add files via upload

nberdy committed:
Update iot-hub-store-data-in-azure-table-storage.md

Updating tutorial to account for route to storage feature

nberdy committed:
Add files via upload

nberdy committed:
Update iot-hub-store-data-in-azure-table-storage.md

zhouwangzw committed:
fix link

rothja committed:
Updating FAQ adding resources and fixing answers

rothja committed:
Minor fix

alkohli committed:
added procedure to install maintenance mode updates

rafats committed:
first commit

alkohli committed:
fine tuning 1

alkohli committed:
create include for maintenancemode procedure

alkohli committed:
fixed broken links

rafats committed:
Added the partition images and text

rafats committed:
Fixing typos

rafats committed:
Fixing typo 2

RenaShahMSFT committed:
InitialSkeleton

RenaShahMSFT committed:
FAQ and Snapshot Create

carolinacmoravia committed:
M12437: Blocked content for localization - LOCBLOCKER

Hi, @novino, 
´ ´ should be replaced for bold type characters to make the content localizable.
Could you help to merge this PR, please?
Many thanks in advance.

Cc: @SunnyDeng

zhouwangzw committed:
fix link

zhouwangzw committed:
fix link

RenaShahMSFT committed:
Snapshot overview and create

RenaShahMSFT committed:
Delete Snapshot

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-prebuilt-domains

RenaShahMSFT committed:
SnapshotOperations

kromerm committed:
New screenshot for ADF V2 Home Page for factory

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

v-geberr committed:
updated luis subscription creation

RenaShahMSFT committed:
Acrolinx Fixes

RenaShahMSFT committed:
Acrolinx fixes

RenaShahMSFT committed:
deleting unwanted files

RenaShahMSFT committed:
Add CLI

zhouwangzw committed:
fix link

v-geberr committed:
edited

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

RenaShahMSFT committed:
Python Page update

v-geberr committed:
edit 2

RenaShahMSFT committed:
Merging and picture border

v-geberr committed:
Luis to LUIS

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

v-geberr committed:
new naming standard

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

v-geberr committed:
blur keys

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

RenaShahMSFT committed:
Addressing George Wallace comments

@georgewallace  Thanks for the review. Feedback is addressed with this publish.

RenaShahMSFT committed:
Fixing Acrolynx

RenaShahMSFT committed:
removing unnecessary lines

RenaShahMSFT committed:
FAQ Cleanup

RenaShahMSFT committed:
Self-Review Snapshot Overview

RenaShahMSFT committed:
Updating next steps in each article

v-geberr committed:
image and h2 changes

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-subscriptionkeys

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-prebuilt-domains

v-geberr committed:
Kamran's changes

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-geberr-prebuilt-domains

cjgronlund committed:
remove author-monitor link and text

Commit:
edit pass: service-fabric-concepts-reconfiguration

alkohli committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into maintm

alkohli committed:
tech review from teddy

mimig1 committed:
Cosmos DB: Updated list of IP portal addresses.

BryanLa committed:
updates

Commit:
fixing images to be compliant

cjgronlund committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into adf-shots

rafats committed:
Final edit

alkohli committed:
acrolinx changes

RenaShahMSFT committed:
Addressing George's comments on Overview Doc

@georgewallace  thank you! :)

RenaShahMSFT committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into snapshotrenashOct4

Commit:
updating image

rothja committed:
Updating FAQ with additional question and formatting

BryanLa committed:
updates

rafats committed:
one letter change :)

BryanLa committed:
updates

RenaShahMSFT committed:
Addressing comments on delete snapshot page and updating gray borders and resizing of screenshots

Commit:
Added missing articles.

BryanLa committed:
updates

BryanLa committed:
updates

RenaShahMSFT committed:
Addressing FAQ feedback

RenaShahMSFT committed:
Addressing Tamra's feedback

Addressed all @tamram feedback. Thanks for the feedback Tamra!

zhouwangzw committed:
fix rendering issues, typos, links and using sentense case in heading

zhouwangzw committed:
fix numbering

RenaShahMSFT committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into snapshotrenashOct4

RenaShahMSFT committed:
Consolidating  three articles - Create, delete and work with snapshots into one

RenaShahMSFT committed:
Addressing Tamra's comments

Thanks @tamram  for the thorough review.

RenaShahMSFT committed:
images had gotten accidentally swapped

genlin committed:
add two items

tamram committed:
adding new files articles to TOC

RenaShahMSFT committed:
Merge pull request #22 from tamram/tamram-tryagain

adding new files articles to TOC

mumian committed:
update

MGoedtel committed:
updated TOC for articles not originally included by author

MGoedtel committed:
revised TOC where two articles link from

genlin committed:
correct the toc

MGoedtel committed:
changed name of two sub-items

mumian committed:
add hbase vnet links

Commit:
Adding SSH instructions back

BryanLa committed:
updates

mumian committed:
fix a link

rjagiewich committed:
Merge pull request #24984 from genlin/master201

add two items into TOC

Commit:
Merge pull request #25002 from mani-ramaswamy/master

Adding SSH instructions back

PRMerger20 committed:
Merge pull request #24999 from mumian/HbaseReplicationLink1009

add hbase vnet links

vhorne committed:
fix minor typo

Commit:
freshness review

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

moderakh committed:
review comments addressed

PRMerger19 committed:
Merge pull request #25006 from msebolt/freshness-review-devguide-d2c-guidance

freshness review

rmca14 committed:
Merge pull request #24791 from RenaShahMSFT/snapshotrenashOct4

SnapshotPublicPreview

Ja-Dunn committed:
Merge pull request #24540 from zhouwangzw/zhouwang/doc-review

new speech documentation for review

cynthn committed:
Update toc.yml

Specialized doesn't use an image, removing "image" from the TOC title.

cynthn committed:
Merge pull request #4 from cynthn/cynthn-patch-5

Update toc.yml

v-alje committed:
Merge pull request #25007 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger19 committed:
Merge pull request #24465 from moderakh/patch-11

python sdk 2.2.1 release

Commit:
freshness review

PRMerger15 committed:
Merge pull request #25008 from v-geberr/v-geberr-prebuilt-domains

V geberr prebuilt domains

rwike77 committed:
changed wording

PRMerger14 committed:
Merge pull request #25014 from msebolt/freshness-review-devguide-c2d-guidance

freshness review

Commit:
Revisions

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasgateway

Commit:
Merge pull request #25015 from rwike77/freshness

changed wording

v-thepet committed:
fix one messed up file

johnkemnetz committed:
add new perms

rjagiewich committed:
Merge pull request #24992 from MGoedtel/UpdateTOC1092017

updated TOC for articles not originally included by author

Lisaco88 committed:
Merge pull request #24751 from alkohli/maintm

Added procedure to install maintenance mode updates

ShawnJackson committed:
Update service-fabric-concepts-reconfiguration.md

Lisaco88 committed:
Merge pull request #24858 from kromerm/makromeradf

New screenshot for ADF V2 Home Page for factory

Commit:
Revisions and acrolinx fixes.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasgateway

Lisaco88 committed:
Merge pull request #24991 from mumian/toc1009

update

Commit:
Merge pull request #25019 from johnkemnetz/patch-1

add new perms

Commit:
Acrolinx fix.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasgateway

v-thepet committed:
actually fix

rloutlaw committed:
remove snapshot issues

ktoliver committed:
Merge pull request #24534 from neilpeterson/acs-powershell

ACS PowerShell Reference

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

nberdy committed:
Update iot-hub-store-data-in-azure-table-storage.md

v-thepet committed:
Merge pull request #24758 from rafats/partitions2

Partitions2

v-shils committed:
Merge pull request #24909 from cjgronlund/adf-shots

remove author-monitor link and text

Commit:
Merge pull request #25016 from Minewiskan/aasgateway

Aasgateway

PRMerger19 committed:
Merge pull request #25018 from v-thepet/breakage

production bug: fix one messed up file causing all the broken links

PRMerger15 committed:
Merge pull request #25023 from rloutlaw/java-functions-update

remove snapshot parameters from java functions quickstart

v-shils committed:
Merge pull request #24941 from BryanLa/bryanla-msi-updates

MSI overview table tweaks

ShawnJackson committed:
Update service-fabric-concepts-replica-lifecycle.md

ShawnJackson committed:
Update service-fabric-concepts-replica-lifecycle.md

v-thepet committed:
Merge pull request #24680 from nberdy/patch-1

Update iot-hub-store-data-in-azure-table-storage.md

v-shils committed:
Merge pull request #24920 from dkkapur/master

fixing images to be compliant

v-shils committed:
Merge pull request #24943 from BryanLa/bryanla-msi-known-issues

Add MSI extension export error to MSI known issues

v-thepet committed:
Merge pull request #24681 from rothja/sqlvmlinuxfaq

First draft of SQL VM Linux FAQ

ShawnJackson committed:
Merge pull request #24910 from LoriMerrick/two-service-fabric-concepts2

edit pass: service-fabric-concepts-reconfiguration2

Lisaco88 committed:
Merge pull request #24950 from jimdial/patch-119

Added missing articles.

Lisaco88 committed:
Merge pull request #25005 from vhorne/fix-face

fix minor typo

tfitzmac committed:
added full examples

Commit:
Merge pull request #24912 from MicrosoftDocs/mimig1-patch-1

Cosmos DB: Updated list of IP portal addresses.

v-shils committed:
Merge pull request #25011 from v-geberr/v-geberr-subscriptionkeys

V geberr subscriptionkeys

PRMerger15 committed:
Merge pull request #24806 from carolinacmoravia/patch-397

M12437: Blocked content for localization - LOCBLOCKER

Lisaco88 committed:
Merge pull request #25009 from cynthn/master

Fixing TOC title

PRMerger16 committed:
Merge pull request #25033 from tfitzmac/1009ref

added full examples
